catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -102,6 +119,8 @@
    * @param camelContext the Camel context to use across all Content Directory Monitors. Note that
    *     if Apache changes this ModelCamelContext interface there is no guarantee that whatever DM
    *     is being used (Blueprint in this case) will be updated accordingly.
+   * @param attributeRegistry {@link AttributeRegistry} to use to perform attribute lookup for
+   *     attribute overrides for inplace monitoring
```

Comment:
✏️I know it's not part of your changes, but should `getRouteDefinitions` be `@VisibleForTesting`? Better yet, can it be removed and inlined where it's used in the unit tests?

Reply:
Replaced all calls in test with `camelContext.getRouteDefinitions()`